Description: MetaInfo-Homepage auf querencialinux.org
 <url type="homepage"> nannte das alte private Repository. Das Paket nennt
 die Projektseite; die Korrektur im Quell-Repository folgt nach Alpha 2,
 der Code-Tag dokumente/0.1.2 bleibt unverändert (CLEAN-1 B2).
Author: pottrauschen <147073098+pottrauschen@users.noreply.github.com>
Forwarded: not-yet (Quellkorrektur in querencia-dokumente nach Alpha 2)
Last-Update: 2026-10-21
---
--- a/data/org.querencia.Dokumente.metainfo.xml
+++ b/data/org.querencia.Dokumente.metainfo.xml
@@ -21,7 +21,7 @@
     <p>Password protected documents are supported. Documents are only displayed, never modified or saved.</p>
     <p xml:lang="de">Passwortgeschützte Dokumente werden unterstützt. Dokumente werden nur angezeigt, nie verändert oder gespeichert.</p>
   </description>
-  <url type="homepage">https://querencialinux.org</url>
+  <url type="homepage">https://querencialinux.org</url>
   <launchable type="desktop-id">org.querencia.Dokumente.desktop</launchable>
   <provides>
     <binary>querencia-dokumente</binary>
